README.md

```diff
@@ -1,101 +1,127 @@
+#### Installing python3.6
 ```
-$ source venv/bin/activate
+$ sudo add-apt-repository ppa:deadsnakes/ppa
+$ sudo apt update
+$ sudo apt install python3.6
 ```
+### Setup the local repository
 
-You should see a `(venv)` appear at the beginning of your terminal prompt (in Linux and Mac at least) indicating that you are working inside the virtualenv. Now when you install something:
 ```
-(venv) $ pip install <package>
+$ cd ~/
+$ mkdir Programming
+$ cd Programming
+$ git clone https://github.com/space-concordia-robotics/robotics-prototype robotics-prototype
 ```
-It will get installed in the `venv`, and not conflict with other projects installed system-wide.
+A local repository should now be created. `robotics-prototype` is the root directory for this project.
+
+### Setup [virtualenv](https://docs.python.org/3.6/library/venv.html#modulevenvhttps://virtualenv.pypa.io/en/stable/userguide/)
 
-To leave the virtual environment run:
-Linux/Mac (bash/zsh):
 ```
-(venv) $ deactivate
+$ cd robotics-prototype
+$ virtualenv -p `which python3.6` venv
+$ source venv/bin/activate
 ```
-
+The virtual environment should now be activated.
 ### Install [dependencies](https://pip.pypa.io/en/stable/user_guide/#requirements-files)
-First, ensure you activated your `venv` and you are in the root project directory. Then, install the required packages:
 ```
 (venv) $ pip install -r requirements.txt -r requirements-dev.txt
 ```
-
 ### Setup [setuptools](https://setuptools.readthedocs.io/en/latest/setuptools.html#development-mode)
-When you try to run `pytest` from anywhere inside the project, there's a very good chance you'll get `ModuleNotFoundError` thrown. This is because absolute imports inside each test file won't work if the test is being executed by `pytest` inside the test's own directory (i.e. the `sys.path` variable will only contain the directory where the test file lives, not the root directory). To encourage best practices, and avoid doing dirty/hacky `sys.path` manipulation within each python file, using `setuptools`'s **develop** feature will allow the virtual environment to create temporary dev package installs using "hooks" (eggs) to each path inside package directory as defined by the "name" attribute in `setup.py` (i.e. `robot` module will be available as though it was installed through `pip`).
-
-In other words, this means all imports of modules inside the `robot` directory should be imported with absolute path, e.g. inside `"tests/unit/motor_test.py"`, the `Motor` class can be imported using:
+Still in the root directory,
 ```
-# tests/unit/motor_test.py
-from robot.basestation.Motor import Motor
-...
+(venv) $ python setup.py develop
+(venv) $ pytest
 ```
+Running `pytest` without doing `python setup.py develop` will give an ModuleNotFound error. To read up more on this, click [here]( https://github.com/space-concordia-robotics/robotics-prototype/edit/master/README.md)
```

Comment:
I don't think you are linking to the correct page

Reply:
oh, yeah it should be /tests/README.md , oops